### PR TITLE
Move some URLs to global.csv

### PR DESCRIPTION
**Pull request by xhdix**
Move some URLs from bi.csv and ir.csv to global.csv

### CODE REVIEW

File: lists/global.csv [hunk 1]
```diff
@@ -1234,3 +1234,18 @@ https://pt.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by
 https://mitm.watch/,ANON,Anonymization and circumvention tools,2019-03-20,magma,This site detects encryption-removing MITM using logic from NDSS17 interception paper
 https://credibilitycoalition.org/,CULTR,Culture,2019-04-19,OONI,Reportedly inaccessible on WE in Egypt (but probably worth testing globally)
 https://black.riseup.net/,ANON,Anonymization and circumvention tools,2019-04-26,LEAP,
+http://bitbucket.org/,COMM,E-commerce,2017-03-02,CIPIT,
```
Comment (hellais) [inline lists/global.csv]: We should only add the https version of bitbucket.org

File: lists/global.csv [hunk 2]
```diff
@@ -1234,3 +1234,18 @@ https://pt.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by
...
+http://opera.com/,CULTR,Culture,2017-03-09,CIPIT,
+http://savefrom.net/,FILE,File-sharing,2017-03-02,CIPIT,
+http://stackexchange.com/,HOST,Hosting and Blogging Platforms,2017-03-09,CIPIT,
+http://stackoverflow.com/,COMT,Communication Tools,2017-03-02,CIPIT,
+http://w3schools.com/,CULTR,Culture,2017-03-09,CIPIT,
+http://wikihow.com/,CULTR,Culture,2017-03-02,CIPIT,
+http://windows.com/,COMM,E-commerce,2017-03-09,CIPIT,
+http://xda-developers.com/,HACK,Hacking Tools,2017-03-09,CIPIT,
+http://xe.com/,COMM,E-commerce,2017-03-09,CIPIT,
+http://bintray.com/,FILE,File-sharing,2017-05-16,ASL19,
```
Comment (hellais) [inline lists/global.csv]: Same here

**Comment by xhdix** [comment]
Done.

File: lists/global.csv [hunk 3]
```diff
@@ -1234,3 +1234,16 @@ https://pt.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by
 https://mitm.watch/,ANON,Anonymization and circumvention tools,2019-03-20,magma,This site detects encryption-removing MITM using logic from NDSS17 interception paper
 https://credibilitycoalition.org/,CULTR,Culture,2019-04-19,OONI,Reportedly inaccessible on WE in Egypt (but probably worth testing globally)
 https://black.riseup.net/,ANON,Anonymization and circumvention tools,2019-04-26,LEAP,
+https://doubleclick.net/,COMM,E-commerce,2017-03-02,CIPIT,
+http://googleusercontent.com/,SRCH,Search Engines,2017-03-02,CIPIT,
```
Comment (hellais) [inline lists/global.csv]: I don't see any content at this site.

Comment (xhdix) [reply lists/global.csv]: Ah! I think it changed to:
http://dl.googleusercontent.com/

Comment (xhdix) [reply lists/global.csv]: Should I remove it or change?

Comment (hellais) [reply lists/global.csv]: I get a 404 when accessing dl.googleusercontent.com. I see that several sub-domains of googleusercontent.com are used for various purposes, but can't seem to find the most canonical one.

Maybe we should drop it unless we have a specific subdomain that is deemed important to test?

Comment (xhdix) [reply lists/global.csv]: it's reasonable.

File: lists/global.csv [hunk 4]
```diff
@@ -1234,3 +1234,15 @@ https://pt.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by
 https://mitm.watch/,ANON,Anonymization and circumvention tools,2019-03-20,magma,This site detects encryption-removing MITM using logic from NDSS17 interception paper
 https://credibilitycoalition.org/,CULTR,Culture,2019-04-19,OONI,Reportedly inaccessible on WE in Egypt (but probably worth testing globally)
 https://black.riseup.net/,ANON,Anonymization and circumvention tools,2019-04-26,LEAP,
+https://doubleclick.net/,COMM,E-commerce,2017-03-02,CIPIT,
+https://opera.com/,CULTR,Culture,2017-03-09,CIPIT,
```
Comment (hellais) [inline lists/global.csv]: I think this is the wrong category for this site.

File: lists/global.csv [hunk 5]
```diff
@@ -1234,3 +1234,15 @@ https://pt.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by
...
+https://savefrom.net/,FILE,File-sharing,2017-03-02,CIPIT,
+https://stackexchange.com/,HOST,Hosting and Blogging Platforms,2017-03-09,CIPIT,
+https://stackoverflow.com/,COMT,Communication Tools,2017-03-02,CIPIT,
```
Comment (hellais) [inline lists/global.csv]: These are also not the right categories.

File: lists/global.csv [hunk 6]
```diff
@@ -1234,3 +1234,15 @@ https://pt.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by
...
+https://w3schools.com/,CULTR,Culture,2017-03-09,CIPIT,
+https://wikihow.com/,CULTR,Culture,2017-03-02,CIPIT,
+http://windows.com/,COMM,E-commerce,2017-03-09,CIPIT,
```
Comment (hellais) [inline lists/global.csv]: Maybe let's drop this as it redirects to microsoft.com anyways.

File: lists/global.csv [hunk 7]
```diff
@@ -1234,3 +1234,15 @@ https://pt.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by
...
+https://xda-developers.com/,HACK,Hacking Tools,2017-03-09,CIPIT,
```
Comment (hellais) [inline lists/global.csv]: This seems a bit of a stretch to call it hacking tools.

File: lists/global.csv [hunk 8]
```diff
@@ -1234,3 +1234,15 @@ https://pt.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by
...
+https://xe.com/,COMM,E-commerce,2017-03-09,CIPIT,
+https://bintray.com/,FILE,File-sharing,2017-05-16,ASL19,
+https://bitbucket.org/,HACK,Hacking Tools,2017-05-16,ASL19,
```
Comment (hellais) [inline lists/global.csv]: This should also not be hacking tools.

File: lists/global.csv (hunk 6 above)
Comment (xhdix) [reply lists/global.csv]: Right!

**Comment by hellais** [comment]
Thanks for the fixes and contributing to the test lists! This is good to merge.

**Comment by xhdix** [comment]
You're welcome.

**Pull request merged**